### ISSUE TITLE
Pre-trained models for future generation?

### ISSUE THREAD

**Issue by Yuliang-Zou** [cvondrick/videogan#3]
It seems that the models you released only contain the the two streams part and the discriminator. Can you also share the image encoder part for future generation? (And also the model of baby videos, if possible)

Thanks!

**Comment by cvondrick** [comment]
Yes, should be possible! I can do it next week, if that works for you?

**Comment by Yuliang-Zou** [comment]
Yeah sure. Thank you so much~

**Comment by Yuliang-Zou** [comment]
Hi Carl, any updates on the models?
